### ISSUE TITLE
Opinions on the various dunder directories.

### ISSUE THREAD

**Issue by alloy** [artsy/reaction-force#1]
I kinda like `__tests__` and `__mocks__` etc being next to the source files. I also think we could do the same with storybook stories, i.e. `__stories__` ?

**Comment by orta** [comment]
Yeah, IMO that works (though I'd love to settle on a single underscore ) - we can build a globber + require-r for anything that fits that pattern easily

**Comment by alloy** [comment]
Why the single underscore?

**Comment by orta** [comment]
less intense, I assume its double because people were using a single underscores to raise things to the top of a folder in editors, I've never seen that in Artsy code

![screen shot 2017-02-08 at 10 55 05 am](https://cloud.githubusercontent.com/assets/49038/22744998/106b86bc-eded-11e6-89d5-bd7c0042546a.png)

**Comment by orta** [comment]
Yeah, :D

**Comment by orta** [comment]
Lets do ` __stories__`
